docs/source/reference/oauth.md

```diff
@@ -162,58 +162,58 @@ with a login form.
 
 Third request, following redirect:
 
-- browser->GitHub
-- `GET https://github.com/login/oauth/authorize`
+- Browser -> GitHub
+- `GET https://github.com/login/oauth/authorize`.
```

Comment:
I don't think periods on the end of bullets is desirable when the bullets are fragments and not full sentences (here and all below)